### ISSUE TITLE
Blank screen when using Visual Studio in debug mode WPF apps when backdrop type is set

### ISSUE THREAD

**Issue by meowbie** [MicaForEveryone/MicaForEveryone#278]
[X ] I've read [FAQ](https://github.com/MicaForEveryone/MicaForEveryone/wiki/FAQ).
[X ] I've restarted the affected app after adding rule for it.
[X ] I've checked other issues and this is not a duplicate.

**Describe the bug**
A clear and concise description of what the bug is. Problems with third-party apps is not our priority and prefer finding a workaround on your own. Check our wiki for more help.

When launching any WPF application with Visual Studio in debug mode, the application window turns blank grey. This bug does not happen in 'Run without Debugging' mode.

My guess is this happens due to the toolbar at the top of the window that is only displayed in Debug mode.

Although it is possible to add an exemption rule for the specific process, as I work with a lot of new projects and applications, this becomes extremely inconvenient and will flood my exemption rules.

**Rule**
The rule used for the target application 

Global Rules: backdrop set to Acrylic/Mica/Tabbed causes the issue.

**Target Application**
- Application Name: Visual Studio 2022
- Application Version: 17.4.3

**To Reproduce**

1. Open Visual Studio 2022
2. Create a new .NET Core WPF application (tested with .NET 7)
3. Click the Debug button on the toolbar
4. Observe as the window launches and turns blank grey, instead of displaying a background or whatever contents there should be.

**Expected behavior**

**Screenshots**

![image](https://user-images.githubusercontent.com/77949379/227698450-79ff8a92-2c55-4c3b-8c1f-77e792f9cb22.png)

**Windows Version**
E.g. Windows 10 build 19042, Windows 11 Build 22000, Windows 11 Build 25217 Insider Dev Channel, ...

Windows 11 Version 22H2 Build 22621

**Other tools being used**
[ ] Rectify11
[ ] StartIsBack
[ ] Chrome/Firefox themes or config to add system backdrop
[ ] Add it yourself (Any other tool that you may use)

**Additional context**

If it's not possible to apply titebar effects on the window being debugged, it would be great if there was a way to detect that a process was started by the Visual Studio debugger/compiler and automatically exempt it.

**Comment by xmine64** [comment]
> My guess is this happens due to the toolbar at the top of the window that is only displayed in Debug mode.

You're right. That toolbar uses "Window" as class name. You can add a class rule for that to bypass it.

**Comment by meowbie** [comment]
![image](https://user-images.githubusercontent.com/77949379/227721353-22e05a2f-db81-46e6-a06f-df5f104250ae.png)

Hi, unfortunately that didn't work for me, and the 'Window' class isn't in the list of suggested classes either:
![image](https://user-images.githubusercontent.com/77949379/227722214-d61dc519-8c24-4038-afd5-dd28ba822ae6.png)

**Comment by likena82** [comment]
Bump. This even happens when the Toolbar is deactivated from the settings.
